win32ss/user/ntuser/input.c

```diff
@@ -353,6 +377,36 @@ CreateSystemThreads(UINT Type)
     return 0;
 }
 
+BOOL UserCreateSystemThread(DWORD Type)
+{
+    USER_API_MESSAGE ApiMessage;
+
+    ERR("UserCreateSystemThread: %d\n", Type);
+
+    ASSERT(UserIsEnteredExclusive());
+
+    if (gdwPendingSystemThreads & Type)
+    {
+        ERR("System thread 0x%x already pending for creation\n", Type);
+        return TRUE;
+    }
+
+    /* We can't pass a parameter to the new thread so mark what the new thread needs to do */
+    gdwPendingSystemThreads |= Type;
+
+    /* Ask winsrv to create a new system thread. This new thread will enter win32k again calling UserSystemThreadProc */
+    CsrClientCallServer((PCSR_API_MESSAGE)&ApiMessage,
```

Comment:
This structure will also have a sub-structure (that should be declared in include/reactos/subsys/win/winmsg.h ) that will certainly contain the information for CSRSS for creating the thread within CSRSS or in a different process (e.g. explorer).